docs/getting_started.md

```diff
@@ -129,14 +138,19 @@ You'll need sufficient permissions to register an application with your Azure AD
 
 ## Set up Build, Release Trigger, and Release Multi-Stage Pipeline
 
-Now that you've provisioned all the required Azure resources and service connections, you can set up the pipeline for deploying your machine learning model to production. The pipeline has a sequence of stages for:
+Now that you've provisioned all the required Azure resources and service connections, you can set up the pipelines for deploying your machine learning model to production. 
+
+**There are two main Azure pipelines - one to handle model training and another to handle batch scoring of the model.** 
+
+### **Azure [pipeline](../.pipelines/diabetes_regression-ci.yml) for model training**
```

Comment:
diabetes_regression-ci.yml is model training _and_ deployment

Reply:
Fixed and pushed.